layers.go

```diff
@@ -905,6 +905,7 @@ func (r *layerStore) load(lockedForWriting bool) (bool, error) {
 		// and possibly loadMounts() again.
 	}
 
+	var layersToDeleteIDs []string
```

Comment:
Move this immediately next to `incompleteDeletionErrors`, please — both variables are local to the `errorToResolveBySaving` case, and processed ~together.